packages/components/link-box/src/LinkBox.tsx

```diff
@@ -0,0 +1,27 @@
+import { Slot } from '@spark-ui/slot'
+import { cx } from 'class-variance-authority'
+import { forwardRef } from 'react'
+
+export interface LinkBoxProps extends React.ComponentPropsWithoutRef<'div'> {
+  asChild?: boolean
+}
+
+export const LinkBox = forwardRef<HTMLDivElement, LinkBoxProps>(
+  ({ className, asChild, ...props }, ref) => {
+    const Component = asChild ? Slot : 'div'
+
+    return (
+      <Component
+        ref={ref}
+        data-spark-component="link-box"
+        className={cx(
+          'relative [&_a[href]:not([data-spark-component="link-overlay"])]:relative [&_a[href]:not([data-spark-component="link-overlay"])]:z-raised [&_button]:relative [&_button]:z-raised',
```

Comment:
I don't understand the need for applying z-index ourselves using child selectors.

Reply:
It's because the link overlay's before element is positioned absolutely, so we want the buttons and links to be on top of it.